operations/iuf/workflows/managed_rollout.md

```diff
@@ -0,0 +1,117 @@
+images and CFS configurations.
+
+**`NOTE`** Additional arguments are available to control the behavior of the `managed-nodes-rollout` stage. See the [`managed-nodes-rollout` stage documentation](../stages/managed_nodes_rollout.md) for details and adjust the
+examples below if necessary.
+
+### 2.1 LNet router nodes and gateway nodes
+
+LNet router nodes or gateway nodes should be upgraded before rebooting compute nodes to new images and CFS configurations. Since LNet routers and gateway nodes are examples of application nodes, the instructions in this section
+are the same as in [2.3 Application nodes](#23-application-nodes).
+
+Since LNet router nodes and gateway nodes are not managed by workload managers, the IUF `managed-nodes-rollout` stage cannot reboot them in a controlled manner via the `-mrs stage` argument. The IUF `managed-nodes-rollout` stage
+can reboot LNet router and gateway nodes using the `-mrs reboot` argument, but an immediate reboot of the nodes is likely to be disruptive to users and overall system health and is not recommended. Administrators should determine
+the best approach for rebooting LNet router and gateway nodes outside of IUF that aligns with site preferences.
+
+Once this step has completed:
+
+- Managed LNet router and gateway nodes (if any) have been rebooted to the images and CFS configurations created in previous steps of this workflow
+- Per-stage product hooks have executed for the `managed-nodes-rollout` stage if IUF `managed-nodes-rollout` procedures were used to perform the reboots
+
+### 2.2 Compute nodes
+
+1. Refer to the "Install and Upgrade Framework" section of each individual product's installation documentation to determine if any special actions need to be performed outside of IUF for the `managed-nodes-rollout` stage.
+
+1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on a single node to ensure the node reboots successfully with the desired image and CFS configuration. This node is
+referred to as the "canary node" in the remainder of this section. Use `--limit-managed-rollout` to target the canary node only and use `-mrs reboot` to reboot the canary node immediately.
+
+    (`ncn-m001#`) Execute the `managed-nodes-rollout` stage with a single xname, rebooting the canary node immediately. Replace the example value of `${XNAME}` with the xname of the canary node.
+
+    ```bash
+    XNAME=x3000c0s29b1n0
+    iuf -a "${ACTIVITY_NAME}" -r managed-nodes-rollout --limit-managed-rollout "${XNAME}" -mrs reboot
+    ```
+
+1. Verify the canary node booted successfully with the desired image and CFS configuration.
+
+1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on all nodes, rebooting the nodes in the default staged manner in conjunction with the workload manager.
```

Comment:
I'm not the expert here, but don't we need to do something else to get the nodes to actually reboot with the workload manager when they're idle? Maybe you've already consulted with Jason Sollom on this, just checking.

Reply:
I am not aware of any extra steps needed, but I'll tag @jsollom-hpe for his wisdom.

Reply:
yes, when `--managed-rollout-strategy` is set to `stage`, you'd need a WLM job boundry to trigger the reboot.  when performing a canary exercise, it makes much more sense to use `reboot`.

Reply:
I added a note here about using ```reboot``` instead, if appropriate.